### PR TITLE
large set of changes needed to get our libs to deploy to npm

### PR DESCRIPTION
**Pull request by jonborchardt**
most of these changes dont need to be looked at...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,31 @@
-# Varnish 2.0
+# Varnish2 Monorepo
```
Comment (jonborchardt) [inline README.md:1]: please review all readme/contributing file changes

File: docs/data/material/varnish/colors/ColorValues.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { useTheme, styled } from '@mui/material/styles';
 import Typography from '@mui/material/Typography';
-import { CopyToClipboardButton } from '@allenai/varnish';
+import { CopyToClipboardButton } from '@allenai/varnish2';
```
Comment (jonborchardt) [inline docs/data/material/varnish/colors/ColorValues.js:9]: i had to remove icon from copytoclipboard, i would like to put it back some day if we figure out how to get the build to properly resolve the icons

File: docs/pages/material-ui/varnish/eslint-config-varnish.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
-
 import * as React from 'react';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
-import * as pageProps from 'docs/data/material/varnish/varnish-eslint-config/varnish-eslint-config.md?@mui/markdown';
+import * as pageProps from 'docs/data/material/varnish/eslint-config-varnish/eslint-config-varnish.md?@mui/markdown';
```
Comment (jonborchardt) [inline docs/pages/material-ui/varnish/eslint-config-varnish.js:3]: renamed eslint config to old name, sorry for the time wasting

File: docs/src/modules/components/AppNavDrawer.js [hunk 4]
```diff
@@ -219,7 +220,7 @@ export default function AppNavDrawer(props) {
       <React.Fragment>
         <ToolbarDiv>
           {canonicalAs.startsWith('/material-ui/') && (
-            <ProductIdentifier name="Varnish" metadata="Material UI" />
+            <ProductIdentifier name={`Varnish v${packageJson.version}`} metadata="Material UI" />
```
Comment (jonborchardt) [inline docs/src/modules/components/AppNavDrawer.js:223]: show the current version number in the ui

File: docs/src/pages/premium-themes/onepirate/modules/components/Typography.js [hunk 5]
```diff
@@ -102,8 +102,6 @@ Typography.propTypes = {
     'body2',
     'button',
     'caption',
```
Comment (jonborchardt) [inline docs/src/pages/premium-themes/onepirate/modules/components/Typography.js:104]: whatever i did, seems to work, but removed these

File: packages/varnish/package.json [hunk 6]
```diff
@@ -15,16 +15,22 @@
...
     "type": "git",
-    "url": "https://github.com/allenai/varnish.git",
+    "url": "https://github.com/allenai/varnish-mui.git",
     "directory": "packages/varnish"
   },
   "license": "MIT",
   "homepage": "https://varnishj.allenai.org/material-ui/getting-started/overview/",
   "scripts": {
     "build": "yarn build:legacy && yarn build:modern && yarn build:node && yarn build:stable && yarn build:umd && yarn build:types && yarn build:copy-files",
+    "build:legacy": "node ../../scripts/build.mjs legacy",
```
Comment (jonborchardt) [inline packages/varnish/package.json:25]: go go build

File: packages/varnish/scripts/rollup.config.js [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+import { terser } from 'rollup-plugin-terser';
+import reactIs from 'react-is';
+
+const gzip = promisify(zlib.gzip);
+
+/**
+ * @param {{snapshotPath: string}} options
+ * @returns {import('rollup').Plugin}
+ */
+function sizeSnapshot(options) {
```
Comment (jonborchardt) [inline packages/varnish/scripts/rollup.config.js:19]: file copied

File: packages/varnish/scripts/testModuleAugmentation.js [hunk 8]
```diff
@@ -0,0 +1,65 @@
+const childProcess = require('child_process');
+const { chunk } = require('lodash');
+const glob = require('fast-glob');
+const path = require('path');
+const { promisify } = require('util');
+
+const exec = promisify(childProcess.exec);
+const packageRoot = path.resolve(__dirname, '../');
```
Comment (jonborchardt) [inline packages/varnish/scripts/testModuleAugmentation.js:8]: file copied

File: packages/varnish/src/theme/theme.ts [hunk 9]
```diff
@@ -1,48 +1,8 @@
-import { ThemeOptions, createTheme } from '@mui/material/styles';
+import { createTheme, ThemeOptions } from '@mui/material/styles';
 import { deepmerge } from '@mui/utils';
 
-import {
-  color,
-  color2,
-  darkCategoricalColor,
-  lightCategoricalColor,
```
Comment (jonborchardt) [inline packages/varnish/src/theme/theme.ts:8]: moved to sep file

File: packages/varnish/types/ext.d.ts [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+  Color2Type,
+  LightCategoricalColorType,
+  DarkCategoricalColorType,
+} from '../src/theme/colors';
+import { PaletteExtendedType } from '../src/theme/paletteExtended';
+
+interface VarnishMuiTheme {
+  paletteExtended: PaletteExtendedType;
+  color: ColorType;
+  color2: Color2Type;
```
Comment (jonborchardt) [inline packages/varnish/types/ext.d.ts:12]: moved here

File: babel.config.js [hunk 11]
```diff
@@ -9,7 +9,7 @@ function resolveAliasPath(relativeToBabelConf) {
 }
 
 const defaultAlias = {
-  '@allenai/varnish': resolveAliasPath('./packages/varnish/src'),
+  '@allenai/varnish2': resolveAliasPath('./packages/varnish/src'),
```
Comment (jonborchardt) [inline babel.config.js:12]: many many varnish renames

**Comment by jonborchardt** [comment]
let me know if you have feedback, and ill fix it tomorrow, i want to test this in prod

File: README.md [hunk 12]
```diff
@@ -1,36 +1,31 @@
...
 
-```sh
-yarn && yarn docs:dev
-```
+All Varnish2 code can be found under `packages/varnish` and `packages/eslint-config-varnish`. The corresponding demos for these Varnish2 components can be found under `docs/data/material/varnish` and `docs/pages/material-ui/varnish`. When you add a new component, create a demo page for the component in the docs to test locally.
 
-If you do not have yarn installed, select your OS and follow the instructions on the [Yarn website](https://yarnpkg.com/lang/en/docs/install/#mac-stable).
+**Tip:** To see most relevant source files, filter your searches to: `**/varnish/src/**/*.*, **/eslint-config-varnish/src/**/*.*, **/material/varnish/**/*.*, **/material-ui/varnish/**/*.*`
 
-_DO NOT USE NPM, use Yarn to install the dependencies._
```
Comment (arnavic) [inline README.md:14]: does it matter now if they use npm or yarn?

File: docs/src/pages/premium-themes/onepirate/modules/components/Typography.js (hunk 5 above)
Comment (arnavic) [reply docs/src/pages/premium-themes/onepirate/modules/components/Typography.js:104]: these got auto-generated for me upon building, so i wonder if even though we manually removed them here they will come back with the next checkin that builds the project; we will see :)

File: packages/varnish/README.md [hunk 13]
```diff
@@ -8,12 +8,16 @@ Install the package in your project directory with:
...
+npm install @allenai/varnish2
 
 // with yarn
-yarn add @allenai/varnish
+yarn add @allenai/varnish2
 ```
 
 ## Documentation
 
 [The documentation](https://varnish.apps.allenai.org)
```
Comment (arnavic) [inline packages/varnish/README.md:19]: should this be updated to https://varnish-mui.apps.allenai.org/ ?

**Review by arnavic — COMMENTED** [review]
looks good, let a couple of documentation-related thoughts.

**Pull request merged**